lesson02/assignment/css/style.scss

```diff
@@ -4,3 +4,152 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+// Colors and fonts
+$darkgray: #4d4d4d;
```

Comment:
Usually nice to put these in the variables scss file. Since it would be imported at the top here, you can separate out your code

Of course given this is a small assignment, not necessary, but good practice for large projects